docs/topics/features.md

```diff
@@ -170,6 +170,16 @@ Per default Calico still allows all communication within the cluster. Using Kube
 - [NetworkPolicy Example Walkthrough](https://kubernetes.io/docs/getting-started-guides/network-policy/walkthrough/)
 - [Calico Kubernetes](https://github.com/Azure/aks-engine/blob/master/examples/networkpolicy)
 
+### Calico 3.3 cleanup after upgrading to 3.5
```

Comment:
would https://github.com/Azure/aks-engine/blob/master/examples/networkpolicy/README.md be a better place to put this?

Reply:
also see https://github.com/Azure/aks-engine/blob/master/examples/networkpolicy/README.md#update-guidance-for-clusters-deployed-by-aks-engine-releases-prior-to-0170

Reply:
Ha, I already suggested to Pengfei to move to docs/

But generally open to whatever makes the most sense.